src/inmanta/module.py

```diff
@@ -2301,7 +2301,7 @@ def __init__(self, project: Optional[Project], path: str):
             if os.path.exists(os.path.join(path, ModuleV2.MODULE_FILE)):
                 raise ModuleMetadataFileNotFound(
                     f"Module at {path} looks like a v2 module. Please add it as a v2 requirement with"
-                    " `inmanta modules add <module_name>`"
+                    " `inmanta module add --v2 <module_name>`"
```

Comment:
I wonder, should we just put the correct command in the message? (replace `<module_name>` with the actual module name?_